lib/cog_cmd/ecs/task/list.rb

```diff
@@ -43,6 +44,8 @@ def fetch_task_definitions(client, params, next_token = nil, defs = [])
     end
 
     def process_definition_arns(arns)
+      return unless arns.compact.length > 0
```

Comment:
I meant that `process_definition_arns` would return nil in this case, but a list (of maps?) otherwise. I wanted to be sure that whoever is calling this method can handle that.